admin/data/sql/migrations/20190430141939.do.drop-updatedat-column-triggers-pupil-table.sql

```diff
@@ -0,0 +1,27 @@
+DECLARE @tableName VARCHAR(max)
+DECLARE @sql NVARCHAR(max)
+
+DECLARE cur CURSOR FOR
+    SELECT t.name
+    FROM SYS.COLUMNS c
+    JOIN SYS.TABLES t
+        ON c.OBJECT_ID = t.OBJECT_ID
+    WHERE c.name LIKE '%updatedAt%'
+
+OPEN cur
+
+FETCH NEXT FROM cur INTO @tableName;
+
+WHILE @@fetch_status = 0
+BEGIN
+    SET @sql = 'DROP TRIGGER [mtc_admin].[' + @tableName + 'UpdatedAtTrigger]'
+
+    EXEC(@sql)
+
+    FETCH NEXT FROM cur INTO @tableName;
+END
+
+CLOSE cur
+DEALLOCATE cur
+
+DROP TRIGGER IF EXISTS [mtc_admin].[pupilFeedbackUpdatedAtTrigger]
```

Comment:
@GuyHarwood @jon-shipley this needs to be deleted explicitly as the table `pupilFeedback` is already deleted by a previous migration at this stage.
On the undo part this trigger is recreated but since the table as mentioned does not exist I will have to recreate the trigger so migrating backwards won't fail.

Reply:
i suppose you should do everything defensively and ensure it exists first?

Reply:
@GuyHarwood old migrations are now removed so that pretty much resolves that issue I had as well.